.travis.yml

```diff
@@ -55,6 +55,15 @@ matrix:
       env: TARGET="Linux-X86_64"   LLDB_TESTS="all"  CLANG="1" COVERAGE="1"
     - os: linux
       env: TARGET="Linux-X86_64"   LLDB_TESTS="all"  CLANG="1" RELEASE="1"
+    # Linux-X86_64 (with GDB tests)
```

Comment:
I'm fine with this but I have a question: any reason why we don't just run the gdb tests in the same runs as the lldb tests instead of adding four new build targets?